syntax = "proto3";

import "WordMessage.proto";
import "Empty.proto";

package proto;

service PerspectiveService {
  rpc onWord(WordMessage) returns (Empty) {}
}
